### PR TITLE
[2.5.12] Backport: pass typers to norman stores 

### PR DESCRIPTION
**Pull request by paynejacob**
#35377

original PR: #34847

### CODE REVIEW

File: pkg/auth/api/setup.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"net/http"
 
+	"github.com/rancher/rancher/pkg/api/scheme"
```
Comment (thedadams) [inline pkg/auth/api/setup.go:7]: nit: put this in the group below.

File: pkg/controllers/managementuser/clusterauthtoken/crd.go [hunk 2]
```diff
@@ -3,6 +3,8 @@ package clusterauthtoken
 import (
 	"context"
 
+	"github.com/rancher/rancher/pkg/api/scheme"
```
Comment (thedadams) [inline pkg/controllers/managementuser/clusterauthtoken/crd.go:6]: nit: put this in the group below.

**Review by thedadams — DISMISSED** [review]

**Review by thedadams — APPROVED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Comment by paynejacob** [comment]
waiting for changes from https://github.com/rancher/rancher/pull/35451

**Review by thedadams — APPROVED** [review]

**Review by rmweir — APPROVED** [review]

**Pull request merged**